datarobot_batch_scoring/main.py

```diff
@@ -252,12 +254,18 @@ def main(argv=sys.argv[1:]):
     skip_row_id = parsed_args['skip_row_id']
     output_delimiter = parsed_args.get('output_delimiter')
 
+    if pid and not lid:
```

Comment:
We need than to add some additional effort to differentiate import_id from project_id. If this a looks-like-project-id and no model-id we need break execution and said about the invalid number of arguments (missing LID). If it's looks-like-import-id we need to throw something in UI and then execute it as SSE.

I also didn't quite get the part about the 
>break the notation of supplying model & project id as arguments.

This is _already_ breaking that notation because model_id implicitly become not argument (or not a _required argument_ as argparse want it).